collect.py

```diff
@@ -304,12 +308,12 @@ def count_lines(bug_id):
 
 
 # We restart from the start and follow the saved sequence
-def jump_back(current_path, firefox_driver, chrome_driver, visited_paths, bug):
+def jump_back(current_path, current_path_info, firefox_driver, chrome_driver, visited_paths, bug):
```

Comment:
Let's use `current_path_node` or `current_path_elem`, it makes it more clear that it is a node or element in a current path list.

Reply:
Ah no, this is actually another representation of the path. I guess we need to refactor this a bit otherwise it's confusing.

Reply:
What about removing `current_path` and just using `current_path_info` always instead? (renaming `current_path_info` to `current_path`)

Reply:
I will have to make some changes in previously defined visit and other functions. I will update the PR shortly.

Reply:
@marco-c can I keep `current_path_info` as an object with structure like: 
current_path_info ={'elements' :  [ ], 'chrome_xpath': [ ], 'firefox_xpath': [ ]}
This way I will be able to keep visited_paths as a path of elements only.
If I keep `current_path_info` in the way it is presently (list with three columns same as above), I will have to create a list of elements to check if it exists in `visited_paths`. 
I will do it the way you prefer.

Reply:
The current format looks better to me, instead of three lists.
You can just do `[element for element, firefox_xpath, chrome_xpath in current_path_info]` to retrieve the path of elements.
If you have to do it too many times and the code gets too confusing, then we can do the approach with three lists, but if it's just a couple of times, then the approach with a single list and tuples looks better.